stock_picking_report_valued/models/stock_move_line.py

```diff
@@ -78,3 +81,50 @@ def _compute_sale_order_line_fields(self):
                     "sale_price_total": valued_line.price_total,
                 }
             )
+
+    def _get_aggregated_product_quantities(self, **kwargs):
+        result = super()._get_aggregated_product_quantities(**kwargs)
+        if self.env.context.get("bypass_modification"):
+            return result
+        result = self._get_aggregated_product_quantities_delivery_price(result)
+        return result
+
+    def _get_aggregated_product_quantities_delivery_price(self, aggregated_move_lines):
+        # sale order can have sold products under different prices
+        # currently lines on stock.picking will conceder only first price
+        # so total on sale.order and stock.picking will differ
+        # consider riding of aggregating of lines for stock.picking
```

Comment:
Should we do smth about this, like grouping by `(product, price_unit)`?

Reply:
currently, I am not confident that I hadn't overlooked smth and maybe there is a way to compute it correctly, but that doesn't solve possible issue with rounding in case of partials deliveries